src/app/globalStatusBar/login/stub/credential.js

```diff
@@ -0,0 +1,42 @@
+import submissionParse from '../../../submissions/stub/submission.js'
+
+/**
+ * @name verifyLogin
+ * @desc Verify user permission if logged in using `/users` route of Wafter
+ * @param {Function} func - Function to execute after the promise resolves.
+ * @return {Promise} a fetch() that resolves to the return value of func()
+ * @author minhducsun2002
+ */
+
+async function verifyLogin(func) {
+    return fetch(
+        `http://${window.location.hostname}:${window.location.port}/api/users`
```

Comment:
Shorten this to "/api/users"

Reply:
Address in https://github.com/vnma0/hestia/pull/19/commits/84c0282afababde0e3ad70cf90345d947d22553c.